extras/rya.streams/query-manager/src/main/config/configuration.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<queryManagerConfig>
+    <!-- A list of Query Change Log Sources. A source needs to be defined for
+       - every system that is used to manage Rya Streams Query Change Logs.
+       - The query manager will manage queries for all Rya instances whose
+       - change logs are stored within those sources.
+       -->
+    <queryChangeLogSources>
+        <queryChangeLogSource>
+            <kafka>
+                <hostname>[Kafka Broker Hostname]</hostname>
+                <port>[Kafka Broker Port]</port>
+            </kafka>
+        </queryChangeLogSource>
+    </queryChangeLogSources>
+    
+    <!-- This section defines performance related tuning values. Sensible
+       - default have been provided to simplify configuration. 
+       -->
+    <performanceTunning>
+        <!-- Indicates how frequently the Query Change Logs that are defined
+           - in the sources section of this configuration file are polled for
+           - updates. This value effects how much time may pass between the 
+           - state of a query changing and that change being represented by
+           - the execution engine.
+           -->
+        <queryChanngeLogDiscoveryPeriod>
+            <value>1</value>
+            <!-- Options: {MILLISECONDS, SECONDS, MINUTES} -->
```

Comment:
There are a lot more TimeUnit options

Reply:
I don't think it makes sense to use any of the other options, so I'm choosing to not allow them.

Reply:
Maybe we should put a restriction on value in the schema though. It should only be a positive, non-negative, integer.

Reply:
yeah agreed